airflow/www/static/js/connection_form.js

```diff
@@ -140,6 +140,55 @@ $(document).ready(function () {
     }
   }
 
+  $.fn.serializeObject = function () {
+    const o = {};
+    const a = this.serializeArray();
+    $.each(a, function () {
+      if (this.name === 'csrf_token') {
+        // dont send csrf token
+      } else if (this.name === 'conn_id') {
+        o.connection_id = this.value;
+      } else if (this.value !== '') {
+        o[this.name] = this.value;
+      }
+    });
+    return o;
+  };
+
+  function displayAlert(status, message){
+    const alertClass = status ? 'alert-success' : 'alert-error';
+    let alertBox = $('.container .row .alert');
+    if (alertBox.length) {
+      alertBox.removeClass('alert-success').removeClass('alert-error');
+      alertBox.addClass(alertClass);
+      alertBox.text(message);
+      alertBox.show();
+    } else {
+      alertBox = $('<div class="alert ' + alertClass + '">\n' +
+                   '<button type="button" class="close" data-dismiss="alert">×</button>\n' + message + '</div>');
+
+      $('.container .row').prepend(alertBox).show();
+    }
+  }
+
+  function testConnection() {
+    $.ajax({
+      url: '/api/v1/connections/test',
+      type: 'post',
+      contentType: 'application/json',
+      dataType: 'json',
+      data: JSON.stringify($('form#model_form').serializeObject()),
```

Comment:
Do we need to both serialize and stringify the form data?

Reply:
@bbovenzi I think yes, to match the input expected by the rest api.